### PR TITLE
refactor: allow meter data import from PM data requests

### PR DESCRIPTION
**Pull request by macintoshpie**
#### Any background context you want to provide?
Users of SEED have requested the ability to import meter data from Data Request spreadsheets

#### What's this PR do?
Adds the ability for the existing meter imports flow to import Data Request meter data.

This has two technical implications:
- adding a new `Electricity - Unknown` source type for Meters. Data requests don't specify the type of data
- adding a new meter display type for `Electricity - Unknown`

Note: data requests do not include source meter IDs, so `Unknown` is use for the IDs of meters which come from Data Requests

#### How should this be manually tested?
1. download an Excel workbook from a data request which includes natural gas and/or electricity data. Either use the example file provided by OpenTech, or use SEED's test account and go to reports and for SEED City Test Report click "Download current report in Excel".
2. Make sure one or more of your properties have the appropriate PM ID which will link to the meter data being imported
3. import PM meter data like normally, select the data request file

#### What are the relevant tickets?
#2673 

#### Screenshots (if appropriate)
Screen Shot 2021-06-14 at 5 06 48 PM

### CODE REVIEW

File: seed/lib/superperms/orgs/migrations/0018_update_organization_display_meter_units.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.10 on 2020-03-27 19:13
+
+from django.db import migrations
+
+
+def update_display_meter_units_options(apps, schema_editor):
+    Organization = apps.get_model('orgs', 'Organization')
+    for org in Organization.objects.all():
+        org.display_meter_units['Electric - Unknown'] = 'kWh (thousand Watt-hours)'
+        org.save()
```
Comment (macintoshpie) [inline seed/lib/superperms/orgs/migrations/0018_update_organization_display_meter_units.py:10]: not very efficient, but I didn't want to spend too much time trying to figure out the SQL to make this query happen

Comment (adrian-lara) [reply seed/lib/superperms/orgs/migrations/0018_update_organization_display_meter_units.py:10]: IMO it's probably fine - the magnitude of orgs in production is in the 100s, probably closer to 100 than 1000 even.

File: seed/migrations/0142_auto_20210614_2058.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    operations = [
+        migrations.AlterField(
+            model_name='meter',
+            name='source',
+            field=models.IntegerField(choices=[(1, 'Portfolio Manager'), (2, 'GreenButton'), (3, 'BuildingSync'), (4, 'Portfolio Manager')], default=None, null=True),
+        ),
+        migrations.AlterField(
+            model_name='meter',
+            name='type',
+            field=models.IntegerField(choices=[(1, 'Coal (anthracite)'), (2, 'Coal (bituminous)'), (3, 'Coke'), (4, 'Diesel'), (5, 'District Chilled Water - Absorption'), (6, 'District Chilled Water - Electric'), (7, 'District Chilled Water - Engine'), (8, 'District Chilled Water - Other'), (9, 'District Hot Water'), (10, 'District Steam'), (11, 'Electric - Grid'), (12, 'Electric - Solar'), (13, 'Electric - Wind'), (14, 'Fuel Oil (No. 1)'), (15, 'Fuel Oil (No. 2)'), (16, 'Fuel Oil (No. 4)'), (13, 'Fuel Oil (No. 5 and No. 6)'), (18, 'Kerosene'), (19, 'Natural Gas'), (20, 'Other:'), (21, 'Propane'), (22, 'Wood'), (23, 'Cost'), (24, 'Electric - Unknown')], default=None, null=True),
```
Comment (macintoshpie) [inline seed/migrations/0142_auto_20210614_2058.py:21]: Added source type for PM Data Request (just called 'Portfolio manager' still)
Added meter type for Electricity - Unknown

File: seed/views/v3/import_files.py [hunk 3]
```diff
@@ -904,20 +904,21 @@ def greenbutton_meters_preview(self, request, pk):
...
                 {'status': 'error', 'message': 'Could not find property with pk=' + str(
                     view_id)}, status=status.HTTP_400_BAD_REQUEST)
 
-        meters_parser = MetersParser(org_id, raw_meter_data, source_type=Meter.GREENBUTTON, property_id=property_id)
+        meters_parser = MetersParser.factory(
+            import_file.local_file,
+            org_id,
+            source_type=Meter.GREENBUTTON,
+            property_id=property_id
+        )
```
Comment (macintoshpie) [inline seed/views/v3/import_files.py:919]: lol not much of an improvement here, but I figured if I was creating the factory for both PM file types then I should probably includee greenbutton as well...

**Review by adrian-lara — APPROVED** [review]
Works as expected with the OpenTech file! Code changes look good to me as well

Like the other ticket, we should get this into dev1 and get more folks testing on it sooner than later.

EDIT: worth noting that the GreenButton and original PM (old format) meter imports still work as expected.

File: seed/models/meters.py [hunk 4]
```diff
@@ -60,18 +61,21 @@ class Meter(models.Model):
...
     PORTFOLIO_MANAGER = 1
     GREENBUTTON = 2
     BUILDINGSYNC = 3
+    PORTFOLIO_MANAGER_DATA_REQUEST = 4
 
     SOURCES = (
         (PORTFOLIO_MANAGER, 'Portfolio Manager'),
         (GREENBUTTON, 'GreenButton'),
         (BUILDINGSYNC, 'BuildingSync'),
+        (PORTFOLIO_MANAGER_DATA_REQUEST, 'Portfolio Manager'),
```
[the remark this answers is not in the document]
Comment (macintoshpie) [reply seed/models/meters.py:78]: Yeah, I figure a user doesn't really care to know what the file format was, just that it came from portfolio manager. Open to changing this later if it ends up being confusing

**Pull request merged**